f5lbaasdriver/v2/bigip/constants_v2.py

```diff
@@ -125,20 +147,20 @@
         "VE-10G": {
             "osr": 5,
             "bandwidth": 10000,
+
             "lod": 32,
             "rod": 100000,
             "cod": 5000000,
             "por": 1.00,
             "poc": 1.00
         },
+        # default is only for hardware device
         "default": {
-            "osr": 1,
```

Comment:
Need to keep osr and bandwidth value, because BandwidthCapacityFilter need them when handling unknown bandwidth devices